photon-lib/src/main/java/org/photonvision/simulation/SimCameraProperties.java

```diff
@@ -565,10 +634,10 @@ public Point[] estPixelNoise(Point[] points) {
         for (int i = 0; i < points.length; i++) {
             var p = points[i];
             // error pixels in random direction
-            double error = avgErrorPx + rand.nextGaussian() * errorStdDevPx;
-            double errorAngle = rand.nextDouble() * 2 * Math.PI - Math.PI;
+            double errorX = avgErrorPxX + rand.nextGaussian() * errorStdDevPxX;
+            double errorY = avgErrorPxY + rand.nextGaussian() * errorStdDevPxY;
             noisyPts[i] =
-                    new Point(p.x + error * Math.cos(errorAngle), p.y + error * Math.sin(errorAngle));
+                    new Point(p.x + errorX, p.y + errorY);
```

Comment:
IIRC the previous "average error" from calibration seemed to report the average absolute error. The previous way of estimating noise tries to use both the average (absolute) error and standard deviation which is pretty goofy. I'm not sure these numbers even really make sense to use in the first place, considering they measure solely the reprojection error of the calibrated instrinsics.

Anyway, it probably makes more sense to just consider the corner pixel normally distributed with given standard deviation. Meaning, just remove (deprecate) the average view error from `SimCameraProperties`. This will noticeably lower the noise currently seen by whatever values are used, but the simulation already seemed to overestimate it.

Reply:
I figured I'd try separating x and y noise because I didn't want to assume that the shape of the pixel noise distribution was circular, but with removing support for file based configs there isn't much of a reason to keep it around. 

In this changeset I opted to revert the separation of the two because otherwise we'd be deprecating `setCalibError(double avgErrorPx, double errorStdDevPx)` and introducing `setCalibError(double errorStdDevPxX, double errorStdDevPxY)`. It's always easy to reintroduce at a better time,

Reply:
I think assuming the noise is circular/isotropic is valid here. Again, this is using the reprojection error of the calibration as a baseline for estimating corner detection noise, which would make doing more than just a constant, normal circular distribution difficult. That would imply calculating the noise based on the location in the image, which seems excessive.